### DISCUSSION TITLE
I recommend AIMP for windows and android

### DISCUSSION THREAD

**Discussion by lux5am** [junguler/m3u-radio-music-playlists#23] [General]
First of all thank you for your superb playlist collection. Have been using your playlist for over a year. 
For Windows and Android I recommend to use AIMP player. I think it's started as winamp clone/alternative for windows in early 2000. 
Have been using this app for over a decade mid 2000. It's very stable and has a good support for audio streaming. It can directly import your playlist from github with direct link (raw.githubusercontent.com). the same with the android version. it support multiple playlist on the menu. so it's easier to import many online playlist. it also has an ability to update the playlist online as long as it's imported from a url.

**Comment by junguler** [comment]
thanks for your suggestion, i will add it to the readme the next time it's time to update it

**Reply by lux5am** [reply]
yes, the most suitable app for your playlist. operate like old school audio player(winamp). it doesn't scan the entire disk like other audio player for mobile, just folder(s) you added. it also has an ability to fetch the album cover online for the audio stream(radio) as long as the stream provide a title/album.
![Screenshot_20231016-101748_AIMP](https://github.com/junguler/m3u-radio-music-playlists/assets/110870661/042b0026-05d5-4c49-9c4b-044e64e40a56)

**Reply by junguler** [reply]
that looks quite useful, i'll make sure to links this discussion page in the readme as well
